### PR TITLE
More artists/gallery data and basic tests for content 

### PR DESCRIPTION
**Pull request by jamesmorgan**
* [x] Add more artists data
* [x] Add basic tests for file formats and structure
* [x] Ensure can source locally

### CODE REVIEW

File: test/truffle/ValidateArtistAndGalleryData.spec.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const fs = require('fs');
+const _ = require('lodash');
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .should();
+
+describe('testing gallery and metatdata', function () {
```
Comment (jamesmorgan) [inline test/truffle/ValidateArtistAndGalleryData.spec.js]: @andygray added some basic tests for checking metadata in correct structure, low res image existence, artist code and type code.

File: migrations/3_populate_contracts.js [hunk 2]
```diff
@@ -49,25 +49,43 @@ const loadSeedData = (instance, _curatorAccount, _openingTime) => {
...
-  }));
+  });
+
+  // Each each set of inserts per edition, Promise.each is serial to prevent duplicate transaction issues
+  return Promise.each(populatedMintItems, function (insertsForEditionArray) {
+    console.log(`Minting [${insertsForEditionArray[0].edition}] - total to mint [${insertsForEditionArray.length}]`);
+
+    // insert each series before moving on the to the next one
+    return Promise.map(insertsForEditionArray, function ({tokenUri, edition, costInWei, openingTime}) {
+      return instance.mint(
```
Comment (jamesmorgan) [inline migrations/3_populate_contracts.js]: @andygray Im not using the mint method only as its basically the same, if we are ot drop the edition number then lets ditch the old `mintEdition()`

Comment (andygray) [reply migrations/3_populate_contracts.js]: agree

File: test/truffle/ValidateArtistAndGalleryData.spec.js (hunk 1 above)
Comment (andygray) [reply test/truffle/ValidateArtistAndGalleryData.spec.js]: Nice one

**Pull request merged**